src/columnGrid/ColumnGrid.tsx

```diff
@@ -366,8 +365,8 @@ export const ColumnGrid: React.FC<ColumnGridProps> = React.memo(
         columnWidth={getColumnWidth}
         height={100} //Its going to be ignored due to autoHeight
         autoHeight
-        scrollLeft={scrollLeft}
-        onScroll={onScroll}
+        // scrollLeft={scrollLeft}
```

Comment:
can we remove this commented line or is it necessary?

Reply:
I'll leave it there because I don't know if it's needed for other examples

Reply:
I saw that it was commented on before, so I'll just remove it

Reply:
thanks!